lib/sidekiq/scheduler.rb

```diff
@@ -234,7 +234,9 @@ def self.active_job_enqueue?(klass)
     #
     # @return [Hash]
     def self.prepare_arguments(config)
-      config['class'] = config['class'].constantize if config['class'].is_a?(String)
+      if config['class'].is_a?(String) && Object.const_defined?(config['class'])
+        config['class'] = Object.const_get(config['class'])
+      end
```

Comment:
It seems a good idea to have an explicit flag. Its default value would be `true` if  `defined?(ActiveJob::Enqueing)`. 

Also, it would be great to be able to configure that flag at job level, because activejobs and non-activejobs could coexist in the same project.

Finally, in order to be able to schedule an ActiveJob from project1 and be executed in project2, that job has to be pushed into sidekiq by following the same idea as `ActiveJob::QueueAdapters::SidekiqAdapter`:

```
        {
          'class'        => 'ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper',
          'queue'        => config['queue'],
          'description'  => config['description'],
          'args'         => [{
            'job_class'  => config['class'],
            'job_id'     => SecureRandom.uuid,
            'queue_name' => config['queue'],
            'arguments'  => config['args']
          }]
        }
```

Any thoughts?